managed-services/transactional-email/reference-content/understanding-tem-reputation-score.mdx

```diff
@@ -0,0 +1,75 @@
+Each domain has several reputations associated with different mail receivers. The reputations also depend on the volume of emails you send to specific receivers.
+
+## Factors that impact your domain reputation
+
+Many factors might affect your domain's reputation. The most common are the following:
+
+- Engagement: When determining domain reputation, engagement is one of the principal factors that comes into play. If recipients fail to open your emails or unsubscribe, email providers will interpret it as an indicator of bad reputation and this might impact your score.
+
+- Domain's age: Your domain's age plays a significant role in determining your reputation score. A new domain that sends out a large volume of emails will raise suspicion.
+
+- Emailing practices: The relevance, frequency, and timing of your emails are used to determine the validity of your domain.
+
+- Blacklists: Internet service providers and email providers may add domains and IPs to their blacklists to prevent spamming. Once your domain is blacklisted, your emails will be instantly flagged as spam. Your domain can also be blacklisted if a significant number of your recipients label your messages as spam.
+
+- Spam traps: Your emails might be landing in [spam traps](/managed-services/transactional-email/concepts/#spam-trap) which in turn, negatively impacts your reputation score.
+
+- Engagement: When determining domain reputation, engagement is one of the principal factors that comes into play. If recipients fail to open your emails or unsubscribe, email providers will interpret it as an indicator of bad reputation and this might impact your score.
+
+
+## Emailing best practices to improve your domain reputation
+
+For a good domain reputation, you need to implement the following emailing practices:
+
+- Use the [preheader](/managed-services/transactional-email/concepts/#preheader) to entice your recipients to open your email.
+- Do not use spam phrases in the **subject line** of your emails. Spam words and phrases associated with scams, promises and free gifts arouse suspicion among email service providers.
+- Maintain a 60/40 text-to-image ratio.
+- Send emails with a coherent structure.
+- Keep a clean list of **engaged users** and delete email addresses of recipients that do not engage with your content.
+- Do not purchase an email list as they might include [spam traps](/managed-services/transactional-email/concepts/#spam-trap).
+- Delete emails that have [soft bounced](/managed-services/transactional-email/concepts/#soft-bounce) and [hard bounced](/managed-services/transactional-email/concepts/#hard-bounce).
+- Use **dedicated** or **shared IPs**:
+
+  * If you are sending large volumes of emails (> 300.000 emails/month), consider using a dedicated IP. This ensures that the emails you send are not affected by the reputation of senders using the same IP address.
```

Comment:
```suggestion
  * If you are sending large volumes of emails (> 300 000 emails/month), consider using a dedicated IP. This ensures that the emails you send are not affected by the reputation of senders using the same IP address.
```

Reply:
I can never remember if we said to use a space or a comma as separator, but I don't think it should be a dot